src/EnergyPlus/SteamCoils.cc

```diff
@@ -2105,10 +2093,10 @@ namespace SteamCoils {
         return Capacity;
     }
 
-    int GetTypeOfCoil(EnergyPlusData &state,
-                      int const CoilIndex,         // must match coil types in this module
-                      std::string const &CoilName, // must match coil names for the coil type
-                      bool &ErrorsFound            // set to true if problem
+    CoilControlType GetTypeOfCoil(EnergyPlusData &state,
```

Comment:
Changed this method to return the enum type, which required making a few changes around the code to accept the change.  What's nice though is that in other places in the code, they were already just comparing to the int listing, so it was generally pretty quick to compare to the proper enumeration.